views/components/BarGraphTooltip/index.jsx

```diff
@@ -13,26 +13,32 @@ const BarGraphTooltip = ({
 }) => {
   if (!active || !payload?.length) return null
 
+  const siteName = useSiteName ? label : SITE_NAMES[label]
+  const countsTotal = useSiteName
+    ? studyTotals[label].count
+    : studyTotals[SITE_NAMES[label]].count
```

Comment:
what is the difference in the counts between these 2 attributes?

Reply:
When we display 10+ bars on one graph the label is the short code and not the full name. Recharts under the hood will give us a label but if it's the site's short code then we need to get the site's full name to key in and get the study totals.

It's not the counts, it's recharts.